package-builder/flox-build.mk

```diff
@@ -414,24 +446,30 @@ define BUILD_template =
   .PHONY: build/$(_pname)
   build/$(_pname): $(_result)
```

Comment:
(non-blocking) Are these `build/` prefixed targets still used? I can't see anywhere that they are referenced after the `PACKAGES` change.

Reply:
No I don't think so .. they can probably go, but I might be inclined to keep them as analogs to the corresponding clean/* targets.